### PR TITLE
wip

### PR DESCRIPTION
**Pull request by AbstractFruitFactory**

### CODE REVIEW

File: packages/account/src/signingKeychain.ts [hunk 1]
```diff
@@ -194,12 +195,14 @@ const create = (
...
-			map((hardwareSigningKey: HardwareSigningKeyT) =>
-				SigningKey.fromHDPathWithHWSigningKey({
+			map((hardwareSigningKey: HardwareSigningKeyT) => {
+				const signingKey = SigningKey.fromHDPathWithHWSigningKey({
 					hdPath,
 					hardwareSigningKey,
-				}),
-			),
+				})
+				setActiveSigningKey(signingKey)
```
Comment (Sajjon) [inline packages/account/src/signingKeychain.ts:203]: I dont like hiding this quite important side effect, so can we maybe at least add it as a boolean option flag to `input: DeriveHWSigningKeyInput`, naming it: `alsoSwitchTo` like we have done with all other account derivation functions?

File: packages/hardware-ledger/src/device-connection.ts [hunk 2]
```diff
@@ -59,14 +59,17 @@ export type OpenLedgerConnectionInput = Readonly<{
...
 ): Promise<BasicLedgerTransport> => {
 	if (isLoggingEnabled) {
 		log.debug(`🔌⏱ Looking for (unlocked 🔓) Ledger device to connect to.`)
 	}
 
+	/*
 	const basicLedgerTransport: BasicLedgerTransport = await import(
-		'@ledgerhq/hw-transport-node-hid'
+		// @ts-ignore
+		'@aleworm/hw-transport-node-hid'
```
Comment (Sajjon) [inline packages/hardware-ledger/src/device-connection.ts:72]: maybe some note about `ts-ignore` and note about why we are using a custom package?

**Review by Sajjon — APPROVED** [review]
Some minor comments.

File: packages/hardware-ledger/src/device-connection.ts (hunk 2 above)
Comment (AbstractFruitFactory) [reply packages/hardware-ledger/src/device-connection.ts:72]: this isn't used

File: packages/account/src/signingKeychain.ts (hunk 1 above)
Comment (AbstractFruitFactory) [reply packages/account/src/signingKeychain.ts:203]: done

**Pull request merged**